docs/ashell.md

```diff
@@ -183,6 +184,15 @@ Renames filename1 to filename2.
 
 Clears the terminal screen.
 
+### erase
+
+`erase`
+
+Erase flash file system.
+
+Note: this only works on the Arduino 101 and it will require rebooting the
+device to take affect.
```

Comment:
I believe their FS still has caches of the handles and everything, because I've ran a simple LS command and it's still listing all the files after I call erase.  However, after I reboot the board, it doesn't show up anymore, so maybe it's like stored in RAM or something, or maybe we need to call some FS init api again